### PR TITLE
HTML/CSS & JavaScript capstone project 

### PR DESCRIPTION
**Pull request by kambereBr**
Here are some features implemented and learning purposes reached in this Pull Request:

## A. Learning purpose

1. HTML, CSS & Javascript best practices
2. Bootstrap framework integrated
3. Linters configuration
4. Usage of correct github flow
5. UIs adaptable to different screen sizes using media queries
6. Javascript DOM elements and events manipulation
7. Communicating technical concepts to other technical people through the video presentation.
8. etc

## B. Features implemented

1. Two pages built (Home page and about page)
2. Each of these pages have versions for 2 different screen sizes (Mobile and desktop)
3. Links interactions
4. Mobile menu
5. Dynamic page content using JavaScript variables
6. Personalized content
7. README.md file personalized
8. Attractive look design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+                <a class="nav-link" href="#partners-section">Sponsors</a>
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link">Forum</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
```
Comment (Ofarouq310) [inline index.html]: - [ ] Kindly make sure to follow the design guidelines as much as possible, for this section, kindly address the following:-
    
    - You have to build two navigations as in the actual design and try to make them look as much as possible as the actual ones.
    - The hamburger icon should not be visible in Desktop screen views.

| Your design | Actual design |
| ------------- | ------------- |
| image  | image  |

File: README.md [hunk 2]
```diff
@@ -1,2 +1,200 @@
...
+If you like this project do not hesitate to rate it, Join the community, Spread the word and Donate If you're able.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse team and all my teammate who somehow contributed to this project 
+
```
Comment (Ofarouq310) [inline README.md]: - [ ] Kindly note that you have to give appropriate credit to the author of the design, `Cindy Shin`.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+      </section>
+    </main>
+    <footer class="footer-section footer-section-home-page">
+      <div class="logo-section-footer">
+        <img
+          src="assets/images/reviewme-low-resolution-logo-black-on-transparent-background.png"
+          width="280px"
+          height="80px"
+          alt=""
+        />
```
Comment (Ofarouq310) [inline index.html]: - [ ] Kindly note that using inline styles is prohibited and bad practice to follow, please fix that.

File: about.html [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+            </p>
+          </div>
+
+          <div class="about-logo text-center">
+            <img
+              src="assets/images/reviewme-low-resolution-logo-color-on-transparent-background.png"
+              width="150px"
+              height="150px"
+              alt=""
+            />
```
Comment (Ofarouq310) [inline about.html]: - [ ] Kindly note that using inline styles is prohibited and bad practice to follow, please fix that.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+      </section>
+    </main>
+    <footer class="footer-section footer-section-about">
+      <div class="logo-section-footer">
+        <img
+          src="assets/images/reviewme-low-resolution-logo-color-on-transparent-background.png"
+          class="footer-logo-red"
+          width="280px"
+          height="80px"
+          alt=""
```
Comment (Ofarouq310) [inline about.html]: - [ ] Kindly note that using inline styles is prohibited and bad practice to follow, please fix that.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+        />
+        <img
+          src="assets/images/reviewme-low-resolution-logo-white-on-transparent-background.png"
+          class="footer-logo-transparent"
+          width="280px"
+          height="80px"
+          alt=""
+        />
```
Comment (Ofarouq310) [inline about.html]: - [ ] Kindly note that using inline styles is prohibited and bad practice to follow, please fix that.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+    <main>
+      <section class="banner-section">
+        <div class="background-image"></div>
+        <div class="banner-text-content banner-text-content-home-page">
+          <h1 class="mb-3">Hello Developers!</h1>
+          <h2 class="fw-bold h2-banner-home">
+            ReviewMe, GET INSTANT HELP, <br />
+            MEET PROFESSIONALS AND <br />
+            MOVE 5x FASTER
+          </h2>
```
Comment (Ofarouq310) [inline index.html]: - [ ] This is the biggest title in your design, so that should be the one to use the `h1` for.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="banner-section">
+        <div class="background-image"></div>
+        <div class="banner-text-content banner-text-content-home-page">
+          <h1 class="mb-3">Hello Developers!</h1>
```
Comment (Ofarouq310) [inline index.html]: - [ ] This is not the main title of your page, thus, you should use `h2` or `h3` for this one.

File: about.html [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="banner-section banner-section-about-page">
+        <div class="background-image background-image-about-page"></div>
+        <div class="banner-text-content banner-text-content-about">
+          <h1 class="mb-3 text-center">Hello Developers!</h1>
+          <h2 class="fw-bold text-center">
```
Comment (Ofarouq310) [inline about.html]: - [ ] KIndly delete this since there is no use for it on the `About` page and not even visible there.

File: about.html [hunk 10]
```diff
@@ -0,0 +1,190 @@
...
+            ReviewMe, GET INSTANT HELP, <br />
+            MEET PROFESSIONALS AND <br />
+            MOVE 5x FASTER
+          </h2>
```
Comment (Ofarouq310) [inline about.html]: - [ ] This is the main title of your page, so it has to be written using the `h1` tag.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+              improve the quality of your code. Learn new skills, and give
+              feedback to the software development community.
+            </p>
+          </div>
+          <div class="site-date-div">
+            <p class="text-date">2023.07.05 (Wed.) ~ 07 (Fri.)</p>
+            <p class="text-date-description">Upcoming ReviewMe Hackathon</p>
+          </div>
+        </div>
+      </section>
```
Comment (Ofarouq310) [inline index.html]: - [ ] Kindly make sure to follow the design guidelines as much as possible, for this section, kindly address the following:-

    - You need to use the proper font, and font size to match the actual design.
    -  You need to add the proper paddings, spacings, and margins to adjust the layout.

| Your design | Actual design |
| ------------- | ------------- |
| image  | image |

File: index.html [hunk 12]
```diff
@@ -0,0 +1,208 @@
...
+          </div>
+
+          <div class="buttont-join-reviewme">
+            <a href="#">JOIN ReviewMe</a>
+          </div>
+          <div class="button-all-service">
+            <a href="#">SEE ALL SERVICES</a>
+          </div>
+        </div>
+      </section>
```
Comment (Ofarouq310) [inline index.html]: - [ ] Kindly make sure to follow the design guidelines as much as possible, for this section, kindly address the following:-

    - You need to use the proper font, and font size to match the actual design.
    -  You need to add the proper paddings, spacings, and margins to adjust the layout.
    - The `Service` title should be adjusted and laid out in the middle of the section.
    - You should add the hover effect on each card as shown in the actual design.

| Your design | Actual design |
| ------------- | ------------- |
| image  | image |

File: assets/js/script.js
Comment (Ofarouq310) [inline assets/js/script.js:1]: - [ ] Kindly make sure to follow the design guidelines as much as possible, for this section, kindly address the following:-

    - You need to use the proper font, and font size to match the actual design.
    -  You need to add the proper paddings, spacings, and margins to adjust the layout.
    - The `Reviewers` title should be adjusted and laid out in the middle of the section.

| Your design | Actual design |
| ------------- | ------------- |
| image  | image |

File: index.html [hunk 13]
```diff
@@ -0,0 +1,208 @@
...
+          <h3 class="text-center">Reviewers</h3>
+          <div class="reviewers-line"><hr /></div>
+        </div>
+      </section>
+      <section
+        class="partners-section partners-section-home-page"
+        id="partners-section"
+      >
+        <h3 class="text-center">Partner</h3>
+        <div class="partners-line"><hr /></div>
```
Comment (Ofarouq310) [inline index.html:192]: - [ ] KIndly make your `Partner` title centered.

| Your design | Actual design |
| ------------- | ------------- |
| image  | image

File: about.html [hunk 14]
```diff
@@ -0,0 +1,190 @@
...
+            <p class="mt-3">
+              Please Contact us per Email for any further questions about
+              ReviewMe! <br />
+              <span class="fw-bold text-decoration-underline"
+                >brunokambere@gmail.com</span
+              >
+            </p>
+          </div>
+        </div>
+      </section>
```
Comment (Ofarouq310) [inline about.html]: - [ ] Kindly make sure to follow the design guidelines as much as possible, for this section, kindly address the following:-

    - You need to use the proper font, and font size to match the actual design.
    -  You need to add the proper paddings, spacings, and margins to adjust the layout.

| Your design | Actual design |
| ------------- | ------------- |
| image | image|

File: about.html [hunk 15]
```diff
@@ -0,0 +1,190 @@
...
+          <div class="hackathon-about-section-image-2">
+            <div class="hackathon-background-image"></div>
+            <div class="hackathon-background-color"></div>
+            <div class="hackathon-description">
+              <h4>2023</h4>
+              <p>ReviewMe Hackathon in Goma</p>
+            </div>
+          </div>
+        </div>
+      </section>
```
Comment (Ofarouq310) [inline about.html]: - [ ] Kindly make sure to follow the design guidelines as much as possible, for this section, kindly address the following:-

    - You need to use the proper font, and font size to match the actual design.
    -  You need to add the proper paddings, spacings, and margins to adjust the layout.

| Your design | Actual design |
| ------------- | ------------- |
| image | image |

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @kambereBr, ✋  

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥈

giphy

## Highlights
✅ Outstanding PR title & summary.
✅ Professional README file.
✅ Meaningful commit messages.

## Required Changes ♻️

- [ ] After reading the comments down there and starting to implement the changes, kindly make sure to keep in mind the following:

    - The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.
    - Basic color swatch: #272A31, #EC5242, #D3D3D3.
    - The pages use the same fonts or similar fonts from the original design.
    - Fonts used originally: Cocogoose, Lato.
    - All page elements have similar spacing, borders, shadows, and paddings to the original design.
    - For every comment, make sure to apply the recommendations to the Mobile screen view.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (kambereBr) [reply index.html]: @Ofarouq310, Thanks a lot for the review.

File: about.html [hunk 16]
```diff
@@ -0,0 +1,187 @@
...
+    <main>
+      <section class="banner-section banner-section-about-page">
+        <div class="background-image background-image-about-page"></div>
+        <div class="banner-text-content banner-text-content-about">
+          <h1 class="mb-3 text-center">Hello Developers!</h1>
+          <h2 class="fw-bold text-center">
+            ReviewMe, GET INSTANT HELP, <br />
+            MEET PROFESSIONALS AND <br />
+            MOVE 5x FASTER
+          </h2>
```
Comment (OyePriscilla) [inline about.html]: - [ ] Please as suggested earlier, kindly ensure the you use `<h1>` tag for this title; this is majorly for SEO. 👍

File: about.html [hunk 17]
```diff
@@ -0,0 +1,187 @@
...
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="banner-section banner-section-about-page">
+        <div class="background-image background-image-about-page"></div>
+        <div class="banner-text-content banner-text-content-about">
+          <h1 class="mb-3 text-center">Hello Developers!</h1>
```
Comment (OyePriscilla) [inline about.html]: - [ ] This supposed to be any of the header tag aside `<h1>`, maybe `<h2 .... h6>`. 👍

File: about.html (hunk 16 above)
Comment (OyePriscilla) [inline about.html]: - [ ] Please as suggested earlier, kindly ensure the you use `<h1>` tag for this title; this is majorly for SEO. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @kambereBr 👍, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Designed according to the given pattern ✔️ 
Acklowledged the Author of the design ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 16 above)
Comment (kambereBr) [reply about.html]: @OyePriscilla, Thanks a lot for your review!

**Review by skyv26 — APPROVED** [review]
## Hi @kambereBr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 




## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Successfully deployed 🚀 ✅ 
- Used `gh-pages` for deployment ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well-structured project ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kambereBr** [comment]
Thanks a lot @skyv26 !

**Pull request merged**